fenicsprecice/fenicsprecice.py

```diff
@@ -46,8 +47,8 @@ def __init__(self, adapter_config_filename='precice-adapter-config.json'):
         adapter_config_filename : string
             Name of the JSON adapter configuration file (to be provided by the user)
         """
-
-        self._config = Config(adapter_config_filename)
+        relative_path = os.path.relpath(adapter_config_filename)
+        self._config = Config(relative_path)
```

Comment:
Can we merge this into one command or is it necessary to define the `relative_path` variable?

Reply:
thanks for your comment. It is done via the latest commit :)